corehq/apps/sms/tasks.py

```diff
@@ -68,6 +68,10 @@ def remove_from_queue(queued_sms):
     sms.publish_change()
 
     tags = {'backend': sms.backend_api}
```

Comment:
set to blank value in case it doesn't exist:

```suggestion
    tags = {'backend': sms.backend_api, 'icds_indicator': ''}
```